guardian/pluto-storagetier
# Feature/gp 632 log correlation

## What does this change?

Adds a "correlation ID" to the logs. This links different messages that relate to the same piece of content

## How to test

Run the updated software. You should see an extra UUID in the log-lines, as the last parameter in the square-brackets.
Update Logstash to ingest this value as a field and you'll be able to filter on it

## How can we measure success?

Easier to see what is happening with problematic items
